keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -638,24 +640,201 @@ always dispatched immediately.  Following is how the other requests
 are dispatched at a given apiserver.
 
 The concurrency limit of an apiserver is divided among the non-exempt
-priority levels in proportion to their assured concurrency shares.
-This produces the assured concurrency value (ACV) for each non-exempt
-priority level:
+priority levels, and they can do a limited amount of borrowing from
+each other.
 
-```
-ACV(l) = ceil( SCL * ACS(l) / ( sum[priority levels k] ACS(k) ) )
+Two fields of `LimitedPriorityLevelConfiguration`, introduced in the
+midst of the `v1beta2` lifetime, configure the borrowing.  The fields
+are added in all the versions (`v1alpha1`, `v1beta1`, and `v1beta2`).
+The following display shows the two new fields along with the updated
+description for the `AssuredConcurrencyShares` field, in `v1beta2`.
+
+**Note**: currently this design does not use the `Priority` field for
```

Comment:
Let's get rid of this field now.

If it appears needed, we can always add that later, but until needed, let's remove it to avoid confusion.

Reply:
I just pushed a revision that removes the Priority field.